calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -146,16 +140,15 @@ Before you get started, make sure you have downloaded and configured the [necess
     helm repo update
     ```
 
-1. Install version {{releaseTitle}} of the {{prodname}} operator and custom resource definitions.
-
+1. Create the tigera-operator namespace.
     ```batch
-    helm install calico projectcalico/tigera-operator --version {{releaseTitle}}
+    kubectl create namespace tigera-operator
     ```
 
-1. Finally, add nodes to the cluster.
+1. Install version {{releaseTitle}} of the {{prodname}} operator and custom resource definitions.
 
     ```batch
-    eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --max-pods-per-node 100
+    helm install calico projectcalico/tigera-operator --namespace tigera-operator --version {{releaseTitle}}
```

Comment:
If this is eks specific. And it is for using tigera-operator helm chart. I believe the values file [here](https://github.com/projectcalico/calico/blob/master/charts/tigera-operator/values.yaml#L5) might want to be set to `"EKS"`. In which case you would need to pass in the `--set installation.kubernetesProvider="EKS"` argument to the helm install command.

Reply:
I think we will want to create a values.yaml file and pass it to helm, with something like this:

```yaml
cat > values.yaml <<EOF
installation:
  kubernetesProvider: EKS
  cni:
    type: Calico
  calicoNetwork:
    bgp: Disabled
EOF
```

Reply:
@ctauchen this particular bit of feedback needs to be addressed before merge. I think it's the only outstanding technical feedback.

I think we need the following as part of this step in order to install correctly.:

---

1. Create a values.yaml file to configure Calico as your network provider.

```
cat > values.yaml <<EOF
installation:
  kubernetesProvider: EKS
  cni:
    type: Calico
  calicoNetwork:
    bgp: Disabled
    ipPools: 
    - encapsulation: VXLAN
      cidr: 192.168.0.0/16
EOF
```

1. Install calico using helm

```
helm install calico projectcalico/tigera-operator --namespace tigera-operator --version {{releaseTitle}} -f values.yaml
```